pkg/credentials/credentials.go

```diff
@@ -142,6 +165,9 @@ func GetPluginImage(pluginName string, dpa *oadpv1alpha1.DataProtectionApplicati
 	case common.VeleroPluginForAWS:
 		return getAWSPluginImage(dpa)
 
+	case common.VeleroPluginForLegacyAWS:
```

Comment:
I would refactor this function to receive `plugin oadpv1alpha1.DefaultPlugin` instead of plugin name, as I described my confusion in another comment